docs/install.md

```diff
@@ -50,9 +50,8 @@ You can refer to the dev docs for more infos [on how to use the installer](./dev
 In a nutshell, invoking the `installer` will look something like this:
 
 ```bash
-DASHBOARD_VERSION=v0.8.0
 curl -sL https://raw.githubusercontent.com/tektoncd/dashboard/master/scripts/release-installer | \
-   bash -s -- install $DASHBOARD_VERSION --read-only
+   bash -s -- install latest --read-only
```

Comment:
using `latest` in the walkthroughs is fine and reduces the overhead of keeping the docs up to date, but here in the install docs we should at least give an example of installing a specific version so users know the format that's expected. Having `latest` as the main example is fine, but we should add a note on installing a specific version:

e.g. ```**Note**: to install a specific version: ... `bash -s -- install v0.8.0 --read-only` ...```

Reply:
good point, i added a note. thanks